deploy-argocd-quicklab.sh

```diff
@@ -0,0 +1,56 @@
+API=upi-0.mptest.lab.upshift.rdu2.redhat.com:6443
+KUBEPASS=rSszg-hYQnn-h9zsi-aAoEb
+
+EMP="\e[1;4m"
+NORMAL="\e[0m"
+
+if which kustomize >/dev/null; then
+  kustomize=kustomize
+else
+  kustomize="toolbox run --container of-toolbox-v0.1.0 kustomize"
+fi
+
+if ! $kustomize --help > /dev/null; then
+  echo -e "${EMP}kustomize is not available --> aborting${NORMAL}"
+  exit 1
+fi
+
+echo -e "${EMP}Checking for GPG key${NORMAL}"
+gpg --list-keys john@doe.com || base64 -d < examples/key.asc | gpg --import
+
+oc login -u kubeadmin -p $KUBEPASS $API
+
+echo -e "${EMP}Creating admin user 'myadmin'${NORMAL}"
+htpasswd -nb myadmin foobar68 > /tmp/oc.htpasswd
+oc create secret generic htpass-secret --from-file=htpasswd=/tmp/oc.htpasswd -n openshift-config
+cat <<EOF | oc apply -f -
```

Comment:
Can you please update this with https://github.com/operate-first/continuous-deployment/pull/69 ? There's no need for a custom oauth provider anymore.

Reply:
Well yes, I'd do that. But as you see: 47 vs 69

It would make much more sense to do changes on top of something that is actually in :-(